lib/gst/player/gstplayer-media-info-private.h

```diff
@@ -94,7 +94,11 @@ struct _GstPlayerMediaInfo
   GObject parent;
 
   gchar *uri;
+  gchar *title;
+  gchar *container;
   gboolean seekable;
+  GstTagList *tags;
+  GstBuffer *cover_image;
```

Comment:
Store the GstSample here. You don't need just the buffer but also the caps, and the info struct in the sample contains information if it's the front cover, back cover, etc.